draft-ietf-scitt-architecture.md

```diff
@@ -290,26 +298,26 @@ SCITT provides a standardized way to:
 
 ### Promotion of a Software Component by Multiple Entities
 
-A software component (e.g., a library) released by a certain original producer is becoming popular.
-The released software component is accompanied by a statement of authenticity (e.g., a detached signature).
+A software component (e.g., a library or software product) released by a certain original producer is common practice for both open-source and commercial offerings.
+The released software component is accompanied by a statement of authenticity (e.g., an embedded or a detached signature).
```

Comment:
SCITT doesn't support embedded signatures, as they don't support the multi-party/post-build processes.
```suggestion
The released software component is accompanied by a statement of authenticity (e.g., a detached signature).
```

Reply:
@SteveLasker  you suggest removing embedded signatures. Does this mean that embedded signatures on artifacts are not supported in SCITT?

Reply:
I'm suggesting we remove references to embedded signatures as SCITT doesn't know about them.
The content also conflates a few different concepts of attached/detached payloads, referencing an artifact the statement is about, and attached signatures that are not part of the SCITT Architecture. 

I'm going to suggest the following change.

SCITT doesn't support embedded signatures, as they don't support the multi-party/post-build processes.
```suggestion
The released software component is accompanied by a statement of authenticity.
```

Reply:
@SteveLasker Many post build artifacts, like MSI installation packages contain embedded digital signatures. This change seems to allow either/or embedded/detached authenticity evidence.

Reply:
Let's make this (item 1) in a corresponding consensus-building discussion.

Reply:
Good idea, Henk. No objection.
The SCITT protocol must exhibit a process is indeed designed to ensure trust, as best as possible. We should try to remove as much uncertainty as possible from the protocol so that people will know a SCITT registry is genuinely a source of trustworthy materials

Reply:
I like the @SteveLasker suggestion of terminating sentence at authenticity.

Reply:
I also like the approach of terminating the sentence early. Cut early, cut often!

More seriously, it's an (eg) so...